src/convertToObject.js

```diff
@@ -3,16 +3,35 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  let stylesArray = sourceString.split('\n');
+
+  stylesArray = stylesArray.filter(style => style.length > 3);
```

Comment:
You can split by this separator ';' and use trim() for delete all spaces and '\n'

Reply:
Strange condition, why do you need it? Why 3?

Agree with @jojnajbude, better to separate by `;` and use trim